src/screens/InboundReceiveDetail/index.tsx

```diff
@@ -34,14 +37,18 @@ const InboundReceiveDetail = () => {
     getInternalLocation(location.id);
   }, [shipmentItem]);
 
-  // eslint-disable-next-line complexity
   const onReceive = () => {
     let errorTitle = '';
     let errorMessage = '';
-    if (!Number(state.quantityToReceive)) {
+    if (state.receiveLocation.id === undefined || !state.receiveLocation.id) {
```

Comment:
You can leave just the `!state.receiveLocation.id`, no need to check if `id === undefined` separately